### PR TITLE
Fix/multichain support

### PR DESCRIPTION
**Pull request by 0xArdi**
## Fixes

This PR fixes multiple issues with multichain support. 

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [x] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes
- [x] Public-facing documentation has been updated with the changes affected by this PR. Even if the provided contents are not in their final form, all significant information must be included.
- [x] Any backwards-incompatible/breaking change has been clearly documented in the upgrading document.

### CODE REVIEW

File: packages/valory/skills/transaction_settlement_abci/skill.yaml [hunk 1]
```diff
@@ -140,7 +140,7 @@ models:
       use_slashing: false
       use_termination: false
       validate_timeout: 1205
-      default_chain_name: ethereum
+      default_chain_id: ethereum
```
Comment (0xArdi) [inline packages/valory/skills/transaction_settlement_abci/skill.yaml:143]: breaking

**Review by DavidMinarsch — APPROVED** [review]

File: packages/valory/skills/transaction_settlement_abci/skill.yaml (hunk 1 above)
Comment (DavidMinarsch) [reply packages/valory/skills/transaction_settlement_abci/skill.yaml:143]: So why are we changing this? Chain id is usually an integer. Chain name is the ecosystem

Comment (0xArdi) [reply packages/valory/skills/transaction_settlement_abci/skill.yaml:143]: the ledger connection expects it as `chain_id`. I agree `chain_name` makes more sense.

Comment (DavidMinarsch) [reply packages/valory/skills/transaction_settlement_abci/skill.yaml:143]: ok - well let's open an issue so we don't forget. Shouldn't block us now to move forward.

**Review by angrybayblade — APPROVED** [review]

File: packages/valory/skills/transaction_settlement_abci/skill.yaml (hunk 1 above)
Comment (0xArdi) [reply packages/valory/skills/transaction_settlement_abci/skill.yaml:143]: https://github.com/valory-xyz/open-aea/issues/694

**Pull request merged**